core/src/main/antlr4/org/jdbi/v3/core/internal/lexer/DefineStatementLexer.g4

```diff
@@ -20,9 +8,9 @@ fragment LT: '<' ;
 fragment GT: '>' ;
 fragment NAME: 'a'..'z' | 'A'..'Z' | '0'..'9' | '_';
 
-COMMENT: '/*' .* '*/';
-QUOTED_TEXT: QUOTE (ESCAPE_QUOTE | ~QUOTE)* QUOTE;
-DOUBLE_QUOTED_TEXT: DOUBLE_QUOTE (~DOUBLE_QUOTE)+ DOUBLE_QUOTE;
+COMMENT: '/*' .*? '*/';
+QUOTED_TEXT: QUOTE (ESCAPE_QUOTE | ~'\'')* QUOTE;
```

Comment:
https://stackoverflow.com/questions/16790861/rule-reference-is-not-currently-supported-in-a-set-in-antlr4-grammar